docs/manuals/source/DeveloperGuide/developerFAQ.rst

```diff
@@ -0,0 +1,117 @@
+Developer FAQ
+=============
+
+How to run a job that automatically fails?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+I want to run a job that automatically fails to simulate failing jobs.
+
+To do so, you can use the directive :config:option:`dir/job/ClientRunBeforeJob`.
+This directive launches a script on the client-side (file daemon) by default before running the job.
+set the script to be “/bin/false”
+
+.. code-block:: bareosconfig
+   :caption: Job with client run before job directive
+
+   Job {
+       Name = "backup-bareos-fd"
+       JobDefs = "DefaultJob"
+       Client = "bareos-fd"
+       Client Run Before Job = /bin/false
+   }
+
+or, if you want a job to work properly but fail at the end (e.g: simulate a full backup that did not go well), you can do the following
+
+.. code-block:: bareosconfig
+   :caption: Job with failing runscript at the end
+
+   Job {
+     Name = "backup"
+     JobDefs = DefaultJob
+     RunScript {
+       Command = "/bin/false"
+       failjobonerror = yes
+       RunsWhen = After
+     }
+    }
+
+
+
+I have changes that affect tapes and autochangers, how can I test them?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to test tapes, we use :command:`mhvtl` to simulate the presence of tape and autochanger devices.
+
+Documentation is available at https://www.mhvtl.com/
+
+Some distributions, like openSUSE, offer mhvtl packages that you can download and install using your package manager.
+Otherwise you need to clone the mhvtl repository from <https://github.com/markh794/mhvtl.git>.
```

Comment:
< > enclosing was in case of `blabla <uri>` markdown, here we can remove them
```suggestion
Otherwise you need to clone the mhvtl repository from https://github.com/markh794/mhvtl.git.
```